GeneratorInterface/GenFilters/interface/MCParticlePairFilter.h

```diff
@@ -38,6 +38,9 @@
 namespace edm {
   class HepMCProduct;
 }
+namespace HepMC {
```

Comment:
Sorry about the late reply. Would it be OK if instead of this declaration I moved the included header https://github.com/echapon/cmssw/blob/bf9d278f552c725aa16496ed0a02029303bc8b4e/GeneratorInterface/GenFilters/src/MCParticlePairFilter.cc#L6 from the .cc to the .h?